mysite/profile/tests.py

```diff
@@ -2466,20 +2487,68 @@ def test_actual_deletion_as_paulproteus(self):
 class TestBreakLongWordsFilter(WebTest):
     def test_too_shirt_to_break(self):
         eight_chars = 'abcdefgh'
-        output = mysite.profile.templatetags.profile_extras.break_long_words(
-            eight_chars)
+        output = mysite.profile.templatetags.profile_extras.break_long_words(eight_chars)
         self.assertEqual(output, eight_chars)
 
     def test_simple_break(self):
         nine_chars = 'abcdefghi'
         nine_chars_plus_wbr = u'abcdefgh\u200bi'
-        output = mysite.profile.templatetags.profile_extras.break_long_words(
-            nine_chars)
+        output = mysite.profile.templatetags.profile_extras.break_long_words(nine_chars)
         self.assertEqual(output, nine_chars_plus_wbr)
 
     def test_break_across_tag_boundaries(self):
         nine_chars = 'abc<i>defghi</i>'
         nine_chars_plus_wbr = u'abcdefgh\u200bi'
-        output = mysite.profile.templatetags.profile_extras.break_long_words(
-            nine_chars)
+        output = mysite.profile.templatetags.profile_extras.break_long_words(nine_chars)
         self.assertEqual(output, nine_chars_plus_wbr)
+
+
+class TestFunctions(WebTest):
+
+    def test_url2printably_short_less_cutoff(self):
+        cutoff = 50
+        url = 'https://example.com'
+        self.assertLess(len(url), cutoff)
+        self.assertTrue(mysite.profile.models.url2printably_short(url, cutoff))
+
+    def test_url2printably_short_equal_cutoff(self):
+        cutoff = 50
+        url = 'https://example.com/blogit/peanutbutter/jelly.html'
+        self.assertEqual(len(url), cutoff)
+        self.assertTrue(mysite.profile.models.url2printably_short(url, cutoff))
+
+    def test_url2printably_short_greater_cutoff(self):
+        cutoff = 50
```

Comment:
I guess you could declare cutoff as a static property of the class here so you don't have to declare it 3 times